app/presenters/scanned_book_show_presenter.rb

```diff
@@ -3,8 +3,11 @@ class ScannedBookShowPresenter < CurationConcerns::GenericWorkShowPresenter
 
   def file_presenters
     @generic_files ||= begin
```

Comment:
@terrellt Just curious...I've seen you do this a few times. What's the advantage of `begin` / `end` when there's no `rescue` or `ensure`?